app/fissile.go

```diff
@@ -951,6 +951,14 @@ func (f *Fissile) generateHelmValues(outputDir string, rolesManifest *model.Role
 		_, err = outputFile.Write(buf)
 	}
 	if err == nil {
+		_, err = outputFile.Write([]byte(`kube:
```

Comment:
That's fine; I'll take. At least it is better than what I was able to get.

I still think the literal YAML is much easier to read, but hey, at least this version is type-safe (or is it?). 😄